pkg/controllers/scheduling/metrics.go

```diff
@@ -0,0 +1,106 @@
+package scheduling
+
+import (
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/utils/clock"
+	"time"
+)
+
+const (
+	SchedulingName        = "placement_scheduling"
+	SchedulingSubsystem   = "scheduling"
+	SchedulingDurationKey = "scheduling_duration_seconds"
+	BindDurationKey       = "bind_duration_seconds"
+)
+
+var (
+	schedulingDuration = k8smetrics.NewHistogramVec(&k8smetrics.HistogramOpts{
+		Subsystem:      SchedulingSubsystem,
+		Name:           SchedulingDurationKey,
+		StabilityLevel: k8smetrics.ALPHA,
+		Help:           "How long in seconds schedule a placement.",
+		Buckets:        k8smetrics.ExponentialBuckets(10e-9, 10, 10),
```

Comment:
The exponential buckets look like
```
# TYPE scheduling_bind_duration_seconds histogram
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1e-08"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1e-07"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1e-06"} 0
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-06"} 1
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="9.999999999999999e-05"} 95
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.001"} 97
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.01"} 116
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="0.1"} 126
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="1"} 126
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="10"} 126
scheduling_bind_duration_seconds_bucket{name="placement_scheduling",le="+Inf"} 126
scheduling_bind_duration_seconds_sum{name="placement_scheduling"} 0.32492658600000013
scheduling_bind_duration_seconds_count{name="placement_scheduling"} 126
```
How about linear buckets?

Reply:
seems to me exponential buckets makes more sense? we care about the placement scheduled less then 1 seconds, and then 10 seconds